await is a keyword in Python 3.7+, so rename 'await' method to avoid SyntaxError
see also https://github.com/pymc-devs/pymc/issues/188
author: Kenneth Hoste (HPC-UGent)
--- pymc-2.3.7/pymc/threadpool.py.orig	2020-03-19 17:48:14.572211381 +0100
+++ pymc-2.3.7/pymc/threadpool.py	2020-03-19 17:49:39.573111182 +0100
@@ -343,7 +343,7 @@
             self.main_lock.release()
         self.counter_lock.release()
 
-    def await(self):
+    def waitforit(self):
         self.main_lock.acquire()
         self.main_lock.release()
 
@@ -384,7 +384,7 @@
                         exc_callback=eb,
                         args=args,
                         requestID=id(args)))
-    done_lock.await()
+    done_lock.waitforit()
 
     if exceptions:
         six.reraise(*exceptions[0])
